cmd/proxy/actions/cdn.go

```diff
@@ -1,10 +1 @@
 package actions
```

Comment:
This file will be empty and can be removed completely. I forget if there was a use still for getting fake metadata. I know this function is not used anywhere currently, just wondering if someone needed it.